packages/wonder-blocks-color/index.js

```diff
@@ -4,7 +4,35 @@ import {mix, fade} from "./util/utils.js";
 const offBlack = "#21242c";
 const white = "#ffffff";
 
-const Color = {
+type ColorType = {|
```

Comment:
In retrospect we probably should've used named exports for the color package.  We could change in this in the future.